Compilation inside docker not working for RX 7600

For some reason when I'm compiling the examples inside this [dockerfile](https://github.com/amd/rocm-examples/blob/develop/Dockerfiles/hip-libraries-rocm-ubuntu.Dockerfile). It gives this error:

```
error: Illegal instruction detected: Invalid dpp_ctrl value: broadcasts are not supported on GFX10+
renamable $vgpr0 = V_MOV_B32_dpp $vgpr0(tied-def 0), killed $vgpr1, 322, 15, 15, 0, implicit $exec
1 error generated when compiling for gfx1102.
make[2]: *** [Libraries/hipCUB/device_radix_sort/CMakeFiles/hipcub_device_radix_sort.dir/build.make:75: Libraries/hipCUB/device_radix_sort/CMakeFiles/hipcub_device_radix_sort.dir/main.hip.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:2455: Libraries/hipCUB/device_radix_sort/CMakeFiles/hipcub_device_radix_sort.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```

Compiling directly on my ArchLinux works flawlessly.

Update 1:

I create a pull request solving this issue here #46.

--- a/Dockerfiles/hip-libraries-rocm-ubuntu.Dockerfile
+++ b/Dockerfiles/hip-libraries-rocm-ubuntu.Dockerfile
@@ -24,10 +24,10 @@ ENV LANG en_US.utf8
 
 # Install ROCM HIP and libraries using the installer script
 RUN export DEBIAN_FRONTEND=noninteractive; \
-    wget https://repo.radeon.com/amdgpu-install/5.3/ubuntu/focal/amdgpu-install_5.3.50300-1_all.deb \
+    wget https://repo.radeon.com/amdgpu-install/5.6.1/ubuntu/focal/amdgpu-install_5.6.50601-1_all.deb \
     && apt-get update -qq \
-    && apt-get install -y ./amdgpu-install_5.3.50300-1_all.deb \
-    && rm ./amdgpu-install_5.3.50300-1_all.deb \
+    && apt-get install -y ./amdgpu-install_5.6.50601-1_all.deb \
+    && rm ./amdgpu-install_5.6.50601-1_all.deb \
     && amdgpu-install -y --usecase=hiplibsdk --no-dkms \
     && apt-get install -y libnuma-dev \
     && rm -rf /var/lib/apt/lists/*

